src/main/webapp/definitions/alfresco-search-sql.yaml

```diff
@@ -0,0 +1,316 @@
+swagger: '2.0'
+info:
+  description: |
+    **Search SQL API**
+    A Solr SQL passthrough, this feature provides the ability to use SQL to query Solr.
+  version: '1'
+  title: Alfresco Insight Engine REST API
+basePath: /alfresco/api/-default-/public/search/versions/1/sql
+securityDefinitions:
+  basicAuth:
+    type: basic
+    description: HTTP Basic Authentication
+security:
+  - basicAuth: []
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  '/sql':
+    post:
+      x-alfresco-since: "6.0"
+      tags:
+        - sql
+      summary: Alfresco Insight Engine SQL Passthrough
+      description: |
+        **Note**: this endpoint is available in Alfresco 6.0 and newer versions.
+        This will require Insight Engine and will not work with Alfresco Search Services.
+
+        **You specify all the parameters in this API in a JSON body**,
+        A basic query looks like this:
+
+        ```JSON
+        {
+          "stmt": "select * from alfresco",
+          "locales": ["en_UK"]
+          "timezone": "Europe/London"
```

Comment:
Do we cover local and timezone in our tests? I can't remember it.

Reply:
Yes, in the tas tests

Reply:
Commas are missing in the body